standard/classes.md

```diff
@@ -485,6 +495,8 @@ If the `where` clause for a type parameter includes a constructor constraint (wh
 
 It is a compile-time error for *type_parameter_constraints* having a *primary_constraint* of `struct` to also have a *constructor_constraint*.
```

Comment:
The following added line is a bit repetitious, maybe:

```suggestion
It is a compile-time error for *type_parameter_constraints* having a *primary_constraint* of `struct` or `unmanaged` to also have a *constructor_constraint*.
```

and deleted the added line?